enhancements/ingress/gateway-api-with-cluster-ingress-operator.md

```diff
@@ -245,10 +237,11 @@ API CRDs, installing OpenShift Service Mesh Operator, managing a
 LoadBalancer-type service for a Gateway, and managing DNS for the same.  This
 requires the following changes:
 
-* Refactor the existing service and DNS management logic to enable its re-use.
-* Add a new controller to manage Gateway API CRDs and install and configure Service Mesh Operator.
-* Add a new controller to manage DNS for Gateways.
+* Refactor the existing DNS management logic to enable its re-use.
+* Add a new controller to manage Gateway API CRDs.
+* Add a new controller to install and configure Service Mesh Operator.
```

Comment:
To match up with L274
```suggestion
* Add a new controller to install and configure Service Mesh.
```

Reply:
How about "the OpenShift Service Mesh Operator"?  I think that that is less ambiguous ("configure Service Mesh" might be interpreted as "configure a service mesh").  Changed in https://github.com/openshift/enhancements/compare/213a342986d10d3ebaaf83ee784ca2867d8b0eb7..dffce6339d368666c5acbb4272e29b8cee3166b2.